vm/systemSmartContracts/governance_test.go

```diff
@@ -1751,10 +1762,9 @@ func TestComputeEndResults(t *testing.T) {
 		Veto:    big.NewInt(10),
 		Abstain: big.NewInt(10),
 	}
-	err = gsc.computeEndResults(pass)
-	require.Nil(t, err)
+	passed = gsc.computeEndResults(pass, baseConfig)
+	require.True(t, passed)
```

Comment:
test needed that shows that the value for the not-passed fee was subtracted from the initial proposal fee in case the proposal did not pass + the initial proposal fee is completely returned if the proposal passed

Reply:
done